dev-ruby/hashicorp-checkpoint/hashicorp-checkpoint-0.1.5-r1.ebuild

```diff
@@ -0,0 +1,31 @@
+# Copyright 1999-2021 Gentoo Authors
```

Comment:
Did you see https://bugs.gentoo.org/723820? That should also be addressed.

Reply:
Yes I've seen this one.
I think it is fixed by my upgrade as the test works on my amd64 setup,
so added the Closes tag to solve this issue.